[BUG] 71.0.0 missing setuptools\cli-64.exe

### setuptools version

setuptools==71.0.0

### Python version

Python 3.10

### OS

Windows 64 Bit

### Additional environment information

_No response_

### Description

trying to editable install a package with console_scripts and getting 
error: [Errno 2] No such file or directory: 'C:\\Users\\mhuisman\\Envs\\3.10\\lib\\site-packages\\setuptools\\cli-64.exe'

reverting to setuptools==70.3.0 fixes the issue

### Expected behavior

no error and installs succesfully

### How to Reproduce

sorry, the code is close-sourced / work
but i suspect trying to install any setup.py with console_scripts in entry_points will cause the issue

### Output

```
    running egg_info
    writing src\my_test_framework.egg-info\PKG-INFO
    writing dependency_links to src\my_test_framework.egg-info\dependency_links.txt
    writing entry points to src\my_test_framework.egg-info\entry_points.txt
    writing requirements to src\my_test_framework.egg-info\requires.txt
    writing top-level names to src\my_test_framework.egg-info\top_level.txt
    reading manifest file 'src\my_test_framework.egg-info\SOURCES.txt'
    reading manifest template 'MANIFEST.in'
    writing manifest file 'src\my_test_framework.egg-info\SOURCES.txt'
    running build_ext
    Creating c:\users\mhuisman\envs\3.10\lib\site-packages\my-test-framework.egg-link (link to src)
    Adding my-test-framework 2.0.365+5eb0314 to easy-install.pth file
    Installing gui_test_runner-script.py script to C:\Users\mhuisman\Envs\3.10\Scripts
    error: [Errno 2] No such file or directory: 'C:\\Users\\mhuisman\\Envs\\3.10\\lib\\site-packages\\setuptools\\cli-64.exe'
    [end of output]
```

Attempting to restore `include_package_data` causes two tests to fail (tests designed to check for the exclusion of tests).

```
_____________________________________________________ TestMeta.test_example_file_not_in_wheel _____________________________________________________
[gw5] darwin -- Python 3.12.4 /Users/jaraco/code/pypa/setuptools/.tox/py/bin/python

self = <setuptools.tests.config.test_apply_pyprojecttoml.TestMeta object at 0x109a9b470>
setuptools_wheel = PosixPath('/private/var/folders/f2/2plv6q2n7l932m2x004jlw340000gn/T/pytest-of-jaraco/pytest-90/dist_build/setuptools-71.0.0.post20240717-py3-none-any.whl')

    def test_example_file_not_in_wheel(self, setuptools_wheel):
        """Meta test to ensure auxiliary test files are not in wheel"""
        with ZipFile(setuptools_wheel) as zipfile:
>           assert not any(name.endswith(EXAMPLES_FILE) for name in zipfile.namelist())
E           AssertionError

setuptools/tests/config/test_apply_pyprojecttoml.py:428: AssertionError
____________________________________________________ test_its_own_wheel_does_not_contain_tests ____________________________________________________
[gw11] darwin -- Python 3.12.4 /Users/jaraco/code/pypa/setuptools/.tox/py/bin/python

setuptools_wheel = PosixPath('/private/var/folders/f2/2plv6q2n7l932m2x004jlw340000gn/T/pytest-of-jaraco/pytest-90/dist_build/setuptools-71.0.0.post20240717-py3-none-any.whl')

    def test_its_own_wheel_does_not_contain_tests(setuptools_wheel):
        with ZipFile(setuptools_wheel) as zipfile:
            contents = [f.replace(os.sep, '/') for f in zipfile.namelist()]
    
        for member in contents:
>           assert '/tests/' not in member
E           AssertionError

/Users/jaraco/code/pypa/setuptools/setuptools/tests/test_setuptools.py:309: AssertionError
```

--- a/newsfragments/4475.bugfix.rst
+++ b/newsfragments/4475.bugfix.rst
@@ -1,0 +1,1 @@
+Restored package data that went missing in 71.0. This change also incidentally causes tests to be installed once again.

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -178,9 +178,7 @@ PKG-INFO = "setuptools.command.egg_info:write_pkg_info"
 "dependency_links.txt" = "setuptools.command.egg_info:overwrite_arg"
 
 [tool.setuptools]
-# disabled as it causes tests to be included #2505
-# include_package_data = true
-include-package-data = false
+include-package-data = true
 
 [tool.setuptools.packages.find]
 include = [

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -194,10 +194,6 @@ exclude = [
 ]
 namespaces = true
 
-[tool.setuptools.package-data]
-# ensure that `setuptools/_vendor/jaraco/text/Lorem ipsum.txt` is installed
-"*" = ["*.txt"]
-
 [tool.distutils.sdist]
 formats = "zip"
 

--- a/setuptools/tests/config/test_apply_pyprojecttoml.py
+++ b/setuptools/tests/config/test_apply_pyprojecttoml.py
@@ -12,7 +12,6 @@ import tarfile
 from inspect import cleandoc
 from pathlib import Path
 from unittest.mock import Mock
-from zipfile import ZipFile
 
 import pytest
 from ini2toml.api import LiteTranslator
@@ -421,11 +420,6 @@ class TestMeta:
         """Meta test to ensure tests can run from sdist"""
         with tarfile.open(setuptools_sdist) as tar:
             assert any(name.endswith(EXAMPLES_FILE) for name in tar.getnames())
-
-    def test_example_file_not_in_wheel(self, setuptools_wheel):
-        """Meta test to ensure auxiliary test files are not in wheel"""
-        with ZipFile(setuptools_wheel) as zipfile:
-            assert not any(name.endswith(EXAMPLES_FILE) for name in zipfile.namelist())
 
 
 class TestInteropCommandLineParsing:

--- a/setuptools/tests/test_setuptools.py
+++ b/setuptools/tests/test_setuptools.py
@@ -301,6 +301,7 @@ def test_findall_missing_symlink(tmpdir, can_symlink):
         assert found == []
 
 
+@pytest.mark.xfail(reason="unable to exclude tests; #4475 #3260")
 def test_its_own_wheel_does_not_contain_tests(setuptools_wheel):
     with ZipFile(setuptools_wheel) as zipfile:
         contents = [f.replace(os.sep, '/') for f in zipfile.namelist()]

--- a/setuptools/tests/test_setuptools.py
+++ b/setuptools/tests/test_setuptools.py
@@ -307,3 +307,10 @@ def test_its_own_wheel_does_not_contain_tests(setuptools_wheel):
 
     for member in contents:
         assert '/tests/' not in member
+
+
+def test_wheel_includes_cli_scripts(setuptools_wheel):
+    with ZipFile(setuptools_wheel) as zipfile:
+        contents = [f.replace(os.sep, '/') for f in zipfile.namelist()]
+
+    assert any('cli-64.exe' in member for member in contents)

